lizard_auth_server/tests/test_admin.py

```diff
@@ -85,7 +86,128 @@ def test_user_profile_link2(self):
             'href' in self.admin_instance.user_profile_link(self.invitation))
 
 
+class TestOrganisationsMigrationAdmin(TestCase):
+
+    def setUp(self):
+        # Three users
+        self.user1 = factories.UserF()
+        self.user2 = factories.UserF()
+        # Old organisation with two users.
+        self.organisation = factories.OrganisationF()
+        self.user1.user_profile.organisations.add(self.organisation)
+        self.user2.user_profile.organisations.add(self.organisation)
+
+        # Rest of the setup.
+        self.request_factory = RequestFactory()
+        self.some_request = self.request_factory.get('/admin/')
+        site = AdminSite()
+        self.admin_instance = admin.OrganisationAdmin(models.Organisation, site)
+
+        # RequestFactory doesn't support middleware, see
+        # http://stackoverflow.com/a/12011907/27401 for below hack.
+        self.some_request.session = 'session'
+        self.messages = FallbackStorage(self.some_request)
+        self.some_request._messages = self.messages
+
+    def test_test_setup(self):
+        self.assertEquals(models.UserProfile.objects.all().count(), 2)
```

Comment:
Het is een conventie in veel unittest frameworks. Maar de BDFL blijkt geen voorkeur te hebben: https://mail.python.org/pipermail/python-dev/2010-December/106954.html